docs/dapp/tools/localnet.mdx

```diff
@@ -0,0 +1,161 @@
+Base HD Path:   m/44'/60'/0'/0/%d
+
+WARNING: The chain is running in ephemeral mode. State will be lost after restart!
+
+ * Listening on http://localhost:8545 and ws://localhost:8546. Chain ID: 0x5afd
+ * Container start-up took 66 seconds, node log level is set to warn.
+```
+
+Those familiar with local dApp environments will find the output above similar
+to `geth --dev` or `ganache-cli` commands or the `geth-dev-assistant` npm
+package. The [sapphire-localnet] will spin up a private Oasis Network locally,
+generate and populate test accounts and make the following Web3 endpoints
+available for you to use:
+
+- `http://localhost:8545`
+- `ws://localhost:8546`
+
+:::tip
+
+If you prefer using the same mnemonics each time (e.g. for testing purposes)
+or to populate just a single account, use `-to` flag and pass the mnemonics or
+the wallet addresses. By passing the `-test-mnemonic` flag you can fund the
+standard test accounts provided by the `hardhat node` commmand and that are
+typically used for solidity unit tests.
+
+<Tabs>
+<TabItem value="Sapphire">
+
+```sh
+docker run -it -p8544-8546:8544-8546 ghcr.io/oasisprotocol/sapphire-localnet -to "bench remain brave curve frozen verify dream margin alarm world repair innocent" -n3
+docker run -it -p8544-8546:8544-8546 ghcr.io/oasisprotocol/sapphire-localnet -to "0x75eCF0d4496C2f10e4e9aF3D4d174576Ee9010E2,0xbDA5747bFD65F08deb54cb465eB87D40e51B197E"
+docker run -it -p8544-8546:8544-8546 ghcr.io/oasisprotocol/sapphire-localnet -test-mnemonic
+```
+
+</TabItem>
+<TabItem value="Emerald">
+
+```sh
+docker run -it -p8544-8546:8544-8546 ghcr.io/oasisprotocol/emerald-localnet -to "bench remain brave curve frozen verify dream margin alarm world repair innocent" -n3
+docker run -it -p8544-8546:8544-8546 ghcr.io/oasisprotocol/emerald-localnet -to "0x75eCF0d4496C2f10e4e9aF3D4d174576Ee9010E2,0xbDA5747bFD65F08deb54cb465eB87D40e51B197E"
+docker run -it -p8544-8546:8544-8546 ghcr.io/oasisprotocol/emerald-localnet -test-mnemonic
+```
+
+</TabItem>
+</Tabs>
+
+:::
+
+:::note Running on Apple M chips
+
+There is currently no `arm64` build available for M Macs, so you will need to
+force the docker image to use the `linux/x86_64` platform, like this:
+
+<Tabs>
+<TabItem value="Sapphire">
+
+```sh
+docker run -it -p8544-8546:8544-8546 --platform linux/x86_64 ghcr.io/oasisprotocol/sapphire-localnet
+```
+
+</TabItem>
+<TabItem value="Emerald">
+
+```sh
+docker run -it -p8544-8546:8544-8546 --platform linux/x86_64 ghcr.io/oasisprotocol/emerald-localnet
+```
+
+</TabItem>
+</Tabs>
+
+:::
+
+:::danger
+
+[sapphire-localnet] and [emerald-localnet] run in ephemeral mode. Any smart
+contract and wallet balance will be lost after you quit the Docker container!
+
+:::
+
+[sapphire-localnet]: https://github.com/oasisprotocol/oasis-web3-gateway/pkgs/container/sapphire-localnet
+[emerald-localnet]: https://github.com/oasisprotocol/oasis-web3-gateway/pkgs/container/emerald-localnet
```

Comment:
Please also add a section on how to run Localnet with github actions. For example:

```yaml
  jobs:
    services:
      sapphire-localnet-ci:
        image: ghcr.io/oasisprotocol/sapphire-localnet
        ports:
          - 8544:8544
          - 8545:8545
          - 8546:8546
        env:
          OASIS_DEPOSIT_BINARY: /oasis-deposit -n 2
        options: >-
          --rm
          --health-cmd="test -f /CONTAINER_READY"
          --health-start-period=90s
```